app/assets/stylesheets/petitions/views/_shared.scss

```diff
@@ -143,6 +143,10 @@ input.back-page {
   background-position: 5px 0px;
 }
 
+.about-item-petition-scheduled-debate-date {
```

Comment:
Bit verbose. This could be `.about-item-scheduled-date` unless we have other scheduled dates